pkg/probe/exec/exec.go

```diff
@@ -66,6 +68,16 @@ func (pr execProber) Probe(e exec.Cmd) (probe.Result, string, error) {
 			}
 			return probe.Failure, string(data), nil
 		}
+
+		timedoutErr, ok = err.(*TimeoutError)
```

Comment:
names consistency...

```suggestion
		timeoutErr, ok = err.(*TimeoutError)
```

Reply:
Fixed